### DISCUSSION TITLE
How to dynamically update the collection alone in the custom angular component filter

### DISCUSSION THREAD

**Discussion by MohanramG** [ghiscoding/Angular-Slickgrid#915] [Q&A] [answered]
is there any service to dynamically update the collection?

  {
        id: 'Name', name: 'Name', field: 'Name', minWidth: 120, sortable: true, filterable: true
        ,
        filter: {
          model: new CustomAngularComponentFilter(),
          collection: this.Name,
          params: {
            component: FilterCompComponent,
          }
        }
      }

**Comment by ghiscoding** [comment, marked as answer]
I'll say again that I prefer you ask questions on Stack Overflow where more users can answer your question(s) or simply search the Wikis. I'll reply you now but I wish this to be the last time to answer your questions here, please use [Stack Overflow](https://stackoverflow.com/search?tab=newest&q=slickgrid) in the future

- [Collection Watch - Wiki](https://github.com/ghiscoding/Angular-Slickgrid/wiki/Select-Filter#collection-watch)
- or look at all the Examples, your use case is demoed in this Example 4 code

https://github.com/ghiscoding/Angular-Slickgrid/blob/965e9ff893f07fdf7323ad374efdbd0ee6a4858f/src/app/examples/grid-editor.component.ts#L516-L533
